src/index.ts

```diff
@@ -67,8 +66,8 @@ function init(modules: { typescript: typeof ts_module }) {
 
         if(config.mockTypeScriptVersion) {
             mockRequire('typescript', ts);
-            tslint = mockRequire.reRequire('tslint');
         }
+        const tslint = require('tslint')
```

Comment:
@egamma I just tried this quickly - it seems to have a knock on effect, in that `create()` has to become async to use it. I'm not sure how much else this will affect.

Reply:
@andyrooger since the code does not rely on typechecking using a require is fine.

Just to confirm, you have tested that this change has no effect when `mockTypeScriptVersion` is false?

Reply:
@egamma Yep. In my case, using Visual Studio Enterprise:

* `mockTypeScriptVersion: true` => Everything works perfectly, lint errors appear in the IDE as they should
* `mockTypeScriptVersion: false` => The wrong version of typescript is loaded successfully, resulting in lots of bizarre lint errors appearing for things they shouldn't. This is exactly how it works for me in 0.9.8 or 0.9.9 without this PR (and with mockTypeScriptVersion turned off).

In both cases I do see linter errors in the IDE, so I presume the packages have at least loaded successfully.